src/utils/toUuid.js

```diff
@@ -3,12 +3,11 @@ const isUUID = require('./isUUID.js');
 const Errors = require('../Errors');
 
 module.exports = async (input) => {
-  if (typeof input !== 'string') throw new Error(Errors.UUID_NICKNAME_MUST_BE_A_STRING);
+if (typeof input !== 'string') throw new Error(Errors.UUID_NICKNAME_MUST_BE_A_STRING);
   if (isUUID(input)) return input.replace(/-/g, '');
   const res = await fetch(`https://api.mojang.com/users/profiles/minecraft/${input}`);
   const parsedRes = await res.json();
   if (parsedRes.error) {
     throw new Error(Errors.MALFORMED_UUID);
   }
-  return parsedRes.id;
```

Comment:
?

Reply:
that _should_ be there

Reply:
tf

Reply:
i think it was your latest commit that broke something cause the function looked way different before

Reply:
oh yes, tried to resolve conflict but something went wrong

Reply:
there
try catch cuz res.json errors if 204

Reply:
i mean, you could just do `if (!res) throw new Error(Errors.PLAYER_DOES_NOT_EXIST);` before `res.json()`

Reply:
res shouldn't be falsy doe?

Reply:
oh mb, it's an empty res.body and not res itself

Reply:
just use the same as in `<Client>._makeRequest()` ?

  ```
const parsedRes = await res.json().catch(() => {
      throw new Error(Errors.INVALID_RESPONSE_BODY);
    });
```